### DISCUSSION TITLE
Allow selection of multiple discussions for reporting

### DISCUSSION THREAD

**Discussion by Andre601** [community/community#115912] [Discussions] [answered]
### Select Topic Area

Question

### Body

The recent flood of discussion spam (See screenshot below) is frustrating. But what is even more frustrating is the way of reporting these to the repository members.

In order to report, one has to go to each individual discussion, click the `...`, select `Report content`, select `spam` on the left side and click `Report to repository admins`.

This entire process is extremely time-consuming while more and more discussions get added to flood the repository with.
There should be a way to select multiple discussions at once to then all report at the same time. To reduce notifications and effort on a repository admin's end, all these reported disucssions should be send as a single notification for the admin(s) to review.

This would allow us users to more easily report spam content to be dealt with, instead of the continous, slow reporting of each discussion individually, which may also consume more time on the admin's end, as they would need to deal with each report separately, instead of looking at them at the same time.*

*Do note that is speculation on my end, as I haven't gotten any reports for my repositories' discussions, nor even use if this feature is enabled for those, nor how it actually would look like, if a report comes in.
But I believe it is one notification per report in this case.

### Screenshot
![grafik](https://github.com/community/community/assets/11576465/3bfd385c-1e5c-493e-9ccd-77eea6b38db6)

**Comment by 0YJ** [comment]
Yes, it is our mission to maintain a clean page and the purity of the open source community.

**Comment by LiteBrite82** [comment, marked as answer]
This is a topic near and dear to my heart on our own discussion boards. First, big thank you to anyone reading this that reports spam on any discussion board; you are fantastic humans! 

As for the batch selection; it sounds good on the surface, but it also leads to false positives, abuse, and trolling. 

However, here is the good news. On the admin side, once users have made a report, the admins have a simple button that can mass delete a users posts and or block the offending party for any amount of time (usually predetermined in the user agreement). so your one report could have a huge effect. 

So for instance, in the screenshot above, as an admin I only need one report on DanBSpringer to delete every post they made and ban them (in accordance with the user guidelines and agreement).

**Reply by Andre601** [reply]
The main issue I can see with the feature you've mentioned is, that there can be countless users spam-posting discussions. And having to figure out which discussions are from what user to report them individually, without making duplicate reports of the same user is even more time-consuming.

Yes, the mass-report feature would lead to false-positives, abuse and trolling. But be honest here, isn't the current report feature abused too? How many times have you received a false-positive, a troll-report, or similar?

Maybe, with mass-reporting posts, only the user is actually reported with links to their discussions to review?
I don't know the behind-the-scenes here of the admin panel for such reports, but I still believe a mass-report can improve things more than it actually harms.

**Reply by Andre601** [reply]
Also, to further add to this, nothing in the report-modal implies that you as admin can actually delete posts of the user as a whole here, so a user may think they have to report each individual post, not knowing that they can just report one and the admin can remove all of them.

That paired with other downsides like A) a limit selection of reasons for reporting (i.e. is "Spam" the best choice for a scam post? Is "off-topic" really a justifiable reason to report a discussion and not just harsh?) and B) not being able to add additional context like you can do with a direct user-report makes this somewhat more frustrating to use and one of the more limited features here.

**Reply by LiteBrite82** [reply]
@Andre601 
We are on it. See similar post: [How to protect spam bot attacks in Discussion?](https://github.com/orgs/community/discussions/121439#discussion-6575877)
Screenshot:
![Screenshot 2024-05-13 at 5 58 16 PM](https://github.com/community/community/assets/163985914/34fd2e29-ae84-44d9-a8fb-0ea33a0619a2)

**Reply by LiteBrite82** [reply]
@jricciardi

**Comment by abbaphy** [comment]
Dear GitHub Support Team,

I hope this email finds you well. I am writing to you because I am currently experiencing an issue with my GitHub account, and I urgently need assistance to regain access.

for the pass couple weeks, I have been unable to log in to my GitHub account with my usual credentials. Despite multiple attempts, I continue to receive an error message indicating that my login credentials are incorrect. I am certain that I am entering the correct username and password, which leads me to believe that there may be an issue with my account.

I am deeply concerned about the security of my account, as I fear it may have been compromised by unauthorized access. I have reviewed my account settings and attempted to reset my password using the "Forgot password?" feature, but I have not been successful in regaining access.

In light of these circumstances, I kindly request your urgent assistance in resolving this matter and restoring access to my GitHub account. Additionally, if there have been any unauthorized activities or changes associated with my account, I would appreciate any information or guidance on how to address them.

Please find below the necessary details for my GitHub account:

Username: [http://github.com/abubakarharuna10]

Email associated with the account:

I am available to provide any further information or clarification that may be required to assist with the resolution of this issue. Your prompt attention to this matter would be greatly appreciated,

Thank you for your understanding and assistance. I look forward to hearing from you soon.

Best regards,
